frameworks/PHP/one/App/Controllers/IndexController.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace App\Controllers;
+
+use App\Model\Fortune;
+use App\Model\World;
+use One\Http\Controller;
+
+class IndexController extends Controller
+{
+
+    protected function json($data)
+    {
+        $this->response->header('Content-type', 'application/json; charset=UTF-8');
+        return json_encode($data);
+    }
+
+    public function toJson()
+    {
+        return $this->json(['message' => 'Hello, World!']);
+    }
+
+    public function plaintext()
+    {
+        $this->response->header('Content-Type', 'text/plain');
+        return 'Hello, World!';
+    }
+
+    public function db()
+    {
+        return $this->json(World::find(mt_rand(1, 10000)));
+    }
+
+    public function fortunes()
+    {
+        $arr   = Fortune::findAll()->toArray();
+        $rs    = [];
+        $rs[0] = 'Additional fortune added at request time.';
+        foreach ($arr as $item) {
+            $rs[$item['id']] = $item['message'];
+        }
+        asort($rs);
```

Comment:
Please, use objects to maintain the full ORM.
Now it's using arrays.
https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview#fortunes

Reply:
`Fortune::findAll()`  is objects to maintain the full ORM

Reply:
For me, I don't see any object.
All is managed like array and the `asort` only order arrays.

Reply:
`0e78a78` commits , The parameter of `asort` is not `Fortune::findAll()`

Reply:
Using usort directly on the $arr array of objects and removing the $rs key value map would fit the requirement. Here is an example:
https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/PHP/kumbiaphp/bench/app/controllers/fortunes_controller.php#L10